charts/kong/values.yaml

```
@@ -408,8 +407,15 @@ postgresql:
 # -----------------------------------------------------------------------------
 
 waitImage:
-  repository: bash
-  tag: 5
+  # Wait for the database to come online before starting Kong or running migrations
+  # If Kong and its database are deployed within a service mesh that restricts
+  # traffic between pods, this must be disabled
```

I think it's not true that we're solving for meshes that "restrict traffic between pods". Instead, we're allowing the mesh to be enabled before a connection to the database occurs (when the database address is expressed in service mesh terms).
```suggestion
  # If Kong is to access the database through a service mesh that injects a sidecar to
  # Kong's container, this must be disabled. Otherwise there'll be a deadlock:
  # InitContainer waiting for DB access that requires the sidecar, and the sidecar
  # waiting for InitContainers to finish.
```